artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -0,0 +1,756 @@
+        - $ref: '#/components/parameters/x-correlator'
+      responses:
+        "204":
+          description: event subscription deleted
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+        "202":
+          description: Request accepted to be processed. It applies for async deletion process.
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SubscriptionAsync'
+        "400":
+          $ref: '#/components/responses/SubscriptionIdRequired'
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/Generic404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+components:
+  securitySchemes:
+    openId:
+      type: openIdConnect
+      openIdConnectUrl: https://example.com/.well-known/openid-configuration
+  parameters:
+    SubscriptionId:
+      name: subscriptionId
+      in: path
+      description: Subscription identifier that was obtained from the create event subscription operation
+      required: true
+      schema:
+        $ref: '#/components/schemas/SubscriptionId'
+    x-correlator:
+      name: x-correlator
+      in: header
+      description: Correlation id for the different services
+      schema:
+        type: string
+  headers:
+    x-correlator:
+      description: Correlation id for the different services
+      schema:
+        type: string
+  schemas:
+    ErrorInfo:
+      type: object
+      required:
+        - status
+        - code
+        - message
+      properties:
+        status:
+          type: integer
+          description: HTTP response status code
+        code:
+          type: string
+          description: Code given to this error
+        message:
+          type: string
+          description: Detailed error description
+        
+    SubscriptionRequest:
+      description: The request for creating a event-type event subscription
+      type: object
+      required:
+        - sink
+        - protocol
+        - config
+      properties:
+        protocol:
+          $ref: "#/components/schemas/Protocol"
+        sink:
+          type: string
+          format: url
+          description: The address to which events shall be delivered using the selected protocol.
+          example: "https://endpoint.example.com/webhook"
+        sinkCredential:
+          $ref: '#/components/schemas/SinkCredential'
+        types:
+          description: Camara Event types eligible to be delivered by this subscription - Use of array MUST be decided at API project level.
+          type: array
+          items:
+            type: string
+        filters:
```

Comment:
@bigludo7 I could not find the comment where you had mentioned that you are ok to remove filter. So, adding a new comment here. I guess since we are anyways using subscription details which acts as an alternative to filter (but simpler), we wont anyways touch filter now. We can remove it if we think it adds confusion. We can revisit it again in a year or later :)

Reply:
Yes work for me. Just checking - @PedroDiez OK for you?

Reply:
alright, then let's keep it out.

Reply:
resolving this